_posts/languages/php/2000-01-01-basic-auth.md

```diff
@@ -32,52 +32,69 @@ Edit the file `nginx-basic-auth.conf` in this directory with the following conte
 
 1. For the complete website:
 
-```nginx
-auth_basic           "Protected Site";
-auth_basic_user_file "/app/config/htpasswd";
-```
+    ```nginx
+    auth_basic           "Protected Site";
+    auth_basic_user_file "/app/config/htpasswd";
+    ```
 
 2. Part of a website, here everything under `/wp-admin`:
 
-```nginx
-location ~ /wp-admin {
-  auth_basic           "Protected Site";
-  auth_basic_user_file "/app/config/htpasswd";
-}
-```
+    ```nginx
+    location ~ /wp-admin {
+      auth_basic           "Protected Site";
+      auth_basic_user_file "/app/config/htpasswd";
+    }
+    ```
 
 3. Depending on the hostname. Useful if you host a staging and a production
    application on Scalingo and just want to protect the staging application with
    basic auth:
 
-```nginx
-if ($host ~ "app-staging.osc-fr1.scalingo.io" ) {
-    set $auth_basic "Protected Site";
-}
-if ($host ~ "app.osc-fr1.scalingo.io" ) {
-    set $auth_basic off;
-}
-auth_basic $auth_basic;
-auth_basic_user_file /app/config/htpasswd;
-```
+    ```nginx
+    if ($host ~ "app-staging.osc-fr1.scalingo.io" ) {
+        set $auth_basic "Protected Site";
+    }
+    if ($host ~ "app.osc-fr1.scalingo.io" ) {
+        set $auth_basic off;
+    }
+    auth_basic $auth_basic;
+    auth_basic_user_file /app/config/htpasswd;
+    ```
 
-Create the `config/htpasswd` file with the couples user/encrypted password
-using the following command:
+4. Create the couples user/encrypted password and set it on your Scalingo app:
 
-```bash
-htpasswd -c config/htpasswd username
+    ```bash
+    scalingo env-set --app my-app HTPASSWD_CONTENT=`htpasswd -n username`
```

Comment:
nitipck: backtick syntax is considered deprecated (https://unix.stackexchange.com/questions/126927/have-backticks-i-e-cmd-in-sh-shells-been-deprecated)
```suggestion
    scalingo env-set --app my-app HTPASSWD_CONTENT=$(htpasswd -n username)
```

Reply:
Thanks !